hmcts/pre-portal
# S28 2916/mk watch page

### Change description

- Created a test watch page that uses the new mk player
- The player temporarily plays a test recording



> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**

- On the portal go to a recording
- In the url replace `/watch` with `/watch-mk`
- Verify that the media kind player is visible and plays a test recoring
